project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala

```diff
@@ -37,7 +37,12 @@ object Docker {
       label("CLIO_VERSION", version.value)
       expose(8080)
       add(artifact, artifactTargetPath)
-      entryPoint("java", "-jar", artifactTargetPath)
+      /*
```

Comment:
The exec will "replace" the shell with the java process.
Maybe that's what you want though.
Could one also get JAVA_OPTS from the environment and pass them explicitly here?